dist/tools/drone-scripts/build_and_test.sh

```diff
@@ -37,12 +37,34 @@ export -f exec_build_func
 MYTMPDIR=$(mktemp -d)
 trap 'rm -rf "$MYTMPDIR"' EXIT
 
-# Execute all groups in parallel (-k ensures correct ordering of the
-# output)
-parallel -k exec_build_func {} "$@"  ::: static-tests avr8 msp430 x86 arm7 \
-                                         cortex_m0 cortex_m3_1 cortex_m3_2 \
-                                         cortex_m4 \
-|& tee -a "$MYTMPDIR/output.log"
+# Check for PRs and labels
+FULL_CHECK=true
+if [ -z "$FORCE_FULL_CHECK" ]; then
+    if [ $TRAVIS_PULL_REQUEST != false ] && ! [ -z "$TRAVIS_PULL_REQUEST" ]; then
```

Comment:
`s/TRAVIS_/CI_/`
Gives the most "correct" variable names in this context, right?